source/pages/general/about.rst

```diff
@@ -1,40 +1,23 @@
-    :depth: 4
-
-.. _intro-datarepo:
-
-====================
-Introduction to Data Repository
-====================
-
-Data Repository Service is a trusted digital repository that ensures long-term data preservation. The Data Repository provides an an online data publication platform service to share and publish research data. The aim of the service is to provide support for researcher to publish research data and to preserve published datasets for the long-term, thereby satisfying the requirements of publishers, financiers and (Dutch) science organisations regarding data publication. Furthermore, the user is complete freedom to decide on the structure of the published data, attached metadata schemas, community and access levels.
-
-There is a direct connection between the data repository service and other SURF services such as the Data Archive. The data that you publish in the data repository will eventually end up on the data archive for long-term storage and preservation. The repository is tape-aware, that means that data can automatically be put offline when it is not required for a certain amount of time.
-
-This document is the user guide for basic and advanced usage of the data repository service and also provide best practice in preparing your data and using the service.
+**************************
 
-.. _what-data:
+The Data Repository Service is a trusted digital repository that ensures long-term data preservation and provides an online data publication platform service to share and publish research data. The aim of the service is to provide support for researcher to publish research data and to preserve published datasets for the long-term, thereby satisfying the requirements of publishers, financiers and (Dutch) science organisations regarding data publication. Furthermore, the user has complete freedom to decide on the structure, the metadata schemas, the designated community, and access levels of the published data.
 
-====================
-For what data
-====================
+The repository provides access to mostly open access data which can be found and downloaded by anyone interested. You can find the data repository at https://repository.surfsara.nl. For publishing datasets an account is required, please see :ref:`this page <get-access>` for information on how to obtain an account.
 
-The Data Repository service is suitable for publishing research data especially in the field of Big Data sciences. The repository imposes no limits on the data volume and number of files, but does have some guidelines on how to efficiently store your data for publication.
+There is a direct connection between the data repository service and other SURF services such as the Data Archive. All data in the data repository will eventually end up in the data archive for long-term storage and preservation. The repository is tape-aware, this means that data can automatically be put offline when it is not required for a certain amount of time.
```

Comment:
To me, the usage of tape for the SDR is more of an implementation detail. It is not necessarily important information for a user to understand the service or decide if they want to use the service. Perhaps we could just say something along the lines that the SURF data archive is used to ensure the long-term preservation of published datasets?

Reply:
It is important, because it might affect researcher's (automatic) workflows. These need to take into account that data can be offline when using the repository for downloads.